cmake/modules/configuration_files.cmake

```diff
@@ -118,3 +118,4 @@ zephyr_boilerplate_watch(DTC_OVERLAY_FILE)
 
 zephyr_get(EXTRA_CONF_FILE SYSBUILD LOCAL VAR EXTRA_CONF_FILE OVERLAY_CONFIG MERGE REVERSE)
 zephyr_get(EXTRA_DTC_OVERLAY_FILE SYSBUILD LOCAL MERGE REVERSE)
+zephyr_get(DTS_EXTRA_CPPFLAGS SYSBUILD LOCAL MERGE REVERSE)
```

Comment:
blocking comment: documentation at top of file has not been updated

Reply:
added